src/main/java/com/kodilla/ecommercee/GroupController.java

```diff
@@ -1,4 +1,35 @@
 package com.kodilla.ecommercee;
 
+import org.omg.CORBA.PUBLIC_MEMBER;
```

Comment:
A po co to?

Reply:
Racja, usunęłam - w nowym commicie nie ma już tego